internal/app/machined/internal/install/install.go

```diff
@@ -94,6 +95,7 @@ func RunInstallerContainer(disk, platform, ref string, configBytes []byte, reg c
 		"--upgrade=" + upgrade,
 		"--force=" + force,
 		"--zero=" + zero,
+		"--legacy-bios-support=" + legacyBIOSSupport,
```

Comment:
this is super tricky: as old installer doesn't support this flag, it would fail on this option, so e.g. you can't downgrade Talos. or you can't install 0.10 on 0.11 PXE boot environment.

so I think we shouldn't be adding any more flags here.

btw, we now have machine configuration passed in to the installer via stdin, why not use it directly?

Reply:
I missed that. Great idea, updated already to read from the config. Will verify and push